README.md

```diff
@@ -1,5 +1,7 @@
 # PDF Exporter
 
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=aerius_webdocument-exporter-plugin&metric=alert_status)](https://sonarcloud.io/dashboard?id=aerius_webdocument-exporter-plugin)
```

Comment:
The `-plugin` bit should be removed. Doesn't match the project name, also not the `projectKey` as you've defined in the pom.